eng/common/cross/toolchain.cmake

```diff
@@ -138,8 +138,8 @@ function(add_toolchain_linker_flag Flag)
   if (NOT Config STREQUAL "")
```

Comment:
Since this file will be overwrritten by the next helix update, this would need to go into arcade.

(proposal https://github.com/dotnet/arcade/issues/6768 was based on this very practical issue, as I myself including 10s of other people have forgot about it in in various repos on several occasions .. a simple GitHub action can probably help 🙂)

Reply:
I know, I have written that in the description above. We need to get this in very quickly to get it into the RC2, so I am making the same change in arcade and here.

Reply:
ah ok, missed that. 👍
just noticed that most recently bot silently overwrote updates made in eng/common ~10 days ago: https://github.com/dotnet/runtime/commit/a5393215debc0eee3669aa9eaee3fc012c20cc6e. Overall this automation seems semi-complete.